src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -109,6 +104,7 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
     val noOp = Future.successful(())
     val processing = for {
       _ <- processTezosBlocks()
+      _ <- processTezosVotes()
```

Comment:
I did this at the request of @anonymoussprocket's original intention for this to be a separate process from Lorre. At the very least, vote aggregation has to be done after operation collection in "processTezosBlocks".

Reply:
If it's independent, it may well be run whenever it happens.

In any case I'd second the idea of having multiple _independent_ processings.
The main point is to figure out what needs to be absolutely consistent with the fetched blocks, and what might be updated more leniently.
If we can make processing independent of the main loop, we have much less work to do to guarantee consistency of all different information stored in the system, which requires some form of "transaction" logic in the main block-processing loop.
An example are accounts checkpoints.
We've gone to great length to guarantee that no account gets lost if something breaks during processing, and it's an additional burden, mentally and programmatically, to maintain such logic.

Reply:
OK, sounds like we should have a proper discussion about this.

For this PR, I'm fine with proceeding with this bit as-is. We can refactor it later.